lib/fog/vmfusion.rb

```diff
@@ -1,5 +1,5 @@
 require 'fog/core'
-require 'fog/vmfusion/version'
+require File.expand_path('../vmfusion/version', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.